inlang/source-code/editor/src/pages/@host/@owner/@repository/State.tsx

```diff
@@ -306,33 +312,86 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 		}
 	)
 
-	repo()?.errors.subscribe((errors: any) => {
-		setLixErrors(errors)
-	})
+	async function pushChanges(args: {
```

Comment:
@martin-lysk  you moved the fuction up, which make 2 pane diffing impossible for the function, is there any reason, or could you move it back to the old position?

Reply:
it removed the repo argument and make it part of the state it self - i needed to move it at the position to have access to repo()

Reply:
I just double checked we added those lines at the top of the function and removed the repo parameter.

```
const loadedRepo = repo()
if (!loadedRepo) {
	return { error: new PushException("Repo not loaded") }
}
```